radixdlt/babylon-wallet-ios
# [ABW-2168] Bug fix - Show `totalExecutionCost` under "NETWORK EXECUTION" (instead of `executionCost`)

Jira ticket: [ABW-2168](https://radixdlt.atlassian.net/browse/ABW-2168)

## Description
Solves UI bug on iOS were we accidentally displayed `executionCost` instead of `totalExecutionCost` which is correct to show.

## How to test
Use Gumball Club dapp, send a swap tx, check the Advanced Fee view and assert that if you sum up all the parts, the sum eq the TRANSACTION FEE displayed in the bottom.

## Screenshot
![IMG_B2BC6A71C299-1](https://github.com/radixdlt/babylon-wallet-ios/assets/116169792/29392092-39fa-4f45-a196-6ef8549764f7)

## Video
https://github.com/radixdlt/babylon-wallet-ios/assets/116169792/168004ee-4f91-43d4-975f-c495368097cf



## PR submission checklist
- [x] I have tested account to account transfer flow and have confirmed that it works, [TXID here](https://rcnet-v3-dashboard.radixdlt.com/transaction/txid_tdx_e_1g7znax2pahwd62g8ulmnl4kr3gyatv8gk807lzp6kz0ly0mdttqs6rkjdu/details)


[ABW-2168]: https://radixdlt.atlassian.net/browse/ABW-2168?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
